src/components/CartProductCard/styles.module.scss

```diff
@@ -0,0 +1,142 @@
+@import '../../assets/styles/variables';
+@import '../../assets/styles/mixins';
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  min-width: 288px;
+  padding: 24px 16px 16px 16px;
+  border: 1px solid $elements;
+  border-radius: 16px;
+  position: relative;
+
+  @include onTablet {
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 24px;
+  }
+}
+
+.containerDeleteImgTitle {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.containerCancelImg {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 16px;
+}
+
+.containerButtonsPrice {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.deleteButton {
+  cursor: pointer;
+  width: 16px;
+  height: 16px;
+  background-image: url('src/assets/images/icons/close.svg');
+  background-color: transparent;
+  border: none;
+
+  transition:
+    background-image $effectDuration ease,
+    transform $effectDuration ease;
+
+  &:hover {
+    transform: scale(1.4);
+    background-image: url('src/assets/images/icons/close-active.svg');
+  }
+}
+
+.picture {
+  width: 80px;
+  height: 72px;
+  object-fit: contain;
+  transition: transform $effectDuration ease;
+
+  &:hover {
+    transform: scale(1.05);
+  }
+}
+
+.title {
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 21px;
+  color: $primary;
+  margin: 0;
+  padding-left: 16px;
+
+  transition: transform $effectDuration ease;
+
+  @include onTablet {
+    margin-right: 24px;
+  }
+
+  &:hover {
+    transform: scale(1.05);
+  }
+}
+
+.buttons {
+  display: flex;
+  align-items: center;
+  margin-right: 32px;
+
+  @include onTablet {
+    margin-right: 0;
+  }
+}
+
+.quantity {
+  min-width: 32px;
+  text-align: center;
+}
+
+.increase {
+  background-image: url('src/assets/images/icons/plus.svg');
+  width: 32px;
+  height: 32px;
+  cursor: pointer;
+  background-color: $white;
+  background-size: 16px 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  border: 1px solid $icons;
+  border-radius: 50%;
+}
+
+.decrease {
```

Comment:
Even if we take out what I wrote about a button that can be reused, you are literally repeating the same code twice. You can give components two classes and put general styles on one and element-specific styles on the other